src/main/java/bhs/devilbotz/RobotContainer.java

```diff
@@ -75,7 +84,6 @@ public RobotContainer() {
     SmartDashboard.putData(driveTrain);
 
     arm.setDefaultCommand(new ArmIdle(arm));
-    gripper.setDefaultCommand(new GripperIdle(gripper));
```

Comment:
The default gripper command was always triggering, causing the gripper open/close command to fail.

Reply:
We can experiment with cleaning this up Monday when we have the actual robot.  The code as it is seems to do the correct thing.